src/OrbitGl/OrbitApp.cpp

```diff
@@ -2818,6 +2824,7 @@ void OrbitApp::OnTimeRangeSelection(uint64_t min, uint64_t max) {
   SetBottomUpView(GetCaptureData().selection_post_processed_sampling_data());
   SetSamplingReport(&GetCaptureData().selection_callstack_data(),
                     &GetCaptureData().selection_post_processed_sampling_data());
+  data_manager_->SetTimeRangeSelection(min, max);
```

Comment:
OrbitApp::OnTimeRangeSelection should probably also take the `struct TimeRange`. 
Yet, refactoring it would be out of the scope of the PR, so it's up to you whether to fix this or just write a todo. 
Sorry I didn't bring it up earler.

Reply:
done.